Add a highlights section to changelog. (#49501)
CleverRaven/Cataclysm-DDA: data/changelog.txt

# 0.F (2021-06-19)

## Highlights
Nested Containers rationalize inventory management and enable dropping and retrieving go-bags during fights.
Achievements track your deeds and misdeeds across games.
Proficiencies better represent deeper knowledge required for various endeavors, mostly crafting.

> Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment system.

Reviewer: ```suggestion
Bleeding added to both the player and monsters as the first step toward a more comprehensive wound and wound treatment systems.
```

Reviewer 2: Would it be different systems or one system?

Reviewer 2: Would it be different systems or one system?

Reviewer 3: Need to drop the `a` if it's systems.
```suggestion
Bleeding added to both the player and monsters as the first step toward more comprehensive wound and wound treatment systems.
```

Author: It's singular, "wound and wound treatment" "system".

Weariness tracking added to represent longer term physical exhaustion.

> Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.

Reviewer: ```suggestion
Elevated bridges/navigable rivers added, allowing better navigabilty while using boats.
```

Reviewer 3: Combination of this and actual-nh's suggestions: https://github.com/CleverRaven/Cataclysm-DDA/pull/49501#discussion_r661132911

> Elevated Bridges/Navigable Rivers added, allowing better navigabilty while using boats.

Reviewer 3: ```suggestion
Elevated bridges over navigable rivers added, allowing better navigability while using boats.
```

Large scale audit of weapon and armor values for better representativeness and consistency.

> Improved armor handling by seperating ballistic damage into it's own damage type.

Reviewer: ```suggestion
Improved armor handling by separating ballistic damage into its own damage type.
```

Pervasive performance enhancements throughout the game.
Tileset vehicle support for more cohesive vehicle rendering.

> Aftershock changing direction to a total conversion with a new far sci-fi setting set on a frozen world.
> Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, dino-related NPCs with missions and dino locations.

Reviewer 2: ```suggestion
Aftershock changes direction to a total conversion mod with a new far sci-fi setting set on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```
Not sure about "far sci-fi".

Reviewer 4: possibly "far future"?

Reviewer 3: ```suggestion
Aftershock changes direction to a total conversion mod with a new far-future sci-fi setting set on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```

Reviewer 2: ```suggestion
Aftershock changes direction to a total conversion mod with a new far-future setting on a frozen world.
Dinomod added 238 dinosaurs, pterosaurs, mosasaurs, and dino-related NPCs with missions and dino locations.
```

Added many dino features, including zombie, fungal, evolved, bionic, baby, and mutant dino variants.

> Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.

Reviewer 4: ```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update with many new traits called Attunements that switch up gameplay at the endgame.
```

Reviewer 2: "Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

Reviewer 2: "Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame."

Reviewer 3: ```suggestion
Added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```

Reviewer 5: Shouldn't each bullet that applies to a specific mod mention the mod? In the list at the top of this section Dinomod is mentioned for the first point, but I'd expect the two following ones to apply to the same mod only.

Reviewer 3: ```suggestion
Dinomod added many dino interactions, including farming, riding, butchering, cooking, and special attacks.
Magiclysm added a huge content update including many new traits called Attunements that switch up gameplay at the endgame.
```
